tests/product/test_installation.py

```diff
@@ -54,11 +54,12 @@ def test_install_non_root(self):
             sudo -u app-admin tar jxf prestoadmin-*.tar.bz2
             cd prestoadmin
             sudo -u app-admin ./install-prestoadmin.sh
-        """.format(mount_dir=DOCKER_MOUNT_POINT, install_dir=install_dir)
+        """.format(mount_dir=DEFAULT_DOCKER_MOUNT_POINT,
```

Comment:
this should use self.docker_cluster.default_mount_dir
Same for all the other tests in this file.